packages/babel-helper-annotate-as-pure/src/index.js

```diff
@@ -2,13 +2,9 @@ import * as t from "@babel/types";
 
 const PURE_ANNOTATION = "#__PURE__";
 
-const isPureAnnotated = node => {
-  const { leadingComments } = node;
-  if (leadingComments === undefined) {
-    return false;
-  }
-  return leadingComments.some(comment => /[@#]__PURE__/.test(comment.value));
-};
+const isPureAnnotated = ({ leadingComments }) =>
+  leadingComments &&
```

Comment:
@existentialism we should always return `boolean` from this

Reply:
@Andarist good point, ill fix